crates/rolldown/src/utils/renamer.rs

```diff
@@ -179,11 +180,11 @@ impl<'name> Renamer<'name> {
               break;
             }
           },
-          Entry::Occupied(_) => {
+          Entry::Occupied(slot) => {
             // The symbol is already renamed
+            used_canonical_names_for_this_scope.insert(slot.get().clone(), 0);
```

Comment:
this operation seems just insert itself again, you could just remove this line.

Reply:
https://github.com/rolldown/rolldown/pull/3645/files#diff-c87caec33470b59eac43cd6ac05bef999e6a4fa010e716fd3e546971148f01d2R178